docs/tutorial/custom-types.txt

```diff
@@ -4,6 +4,12 @@ Custom Data-Types
 
 .. default-domain:: mongodb
 
+.. note::
+
+   This tutorial explains implementing custom data types using the :php:`MongoDB\\BSON\\Persistable <class.mongodb-bson-persistable>`
+   interface found in the MongoDB extension. Consider using a codec instead to decouple the MongoDB persistence logic
+   from your business logic. See the :doc:`Codec tutorial </tutorial/codecs>` for an example.
```

Comment:
```suggestion
   from your business logic. See the :doc:`Codecs tutorial </tutorial/codecs>` for an example.
```

Matches the actual page title. Alternatively, you can use "codec" here but keep it lowercase so it's clear you're not referring to the page title.